docs/emr/presto-admin-installation-on-emr.rst

```diff
@@ -0,0 +1,32 @@
+=========================
+Presto Admin Installation 
```

Comment:
Right. But I think the reader will be confused when given two links to two different pages, telling them to replace some piece on one page based on information in the other. 

We will get rid of this page altogether if and when we decide to do away completely with the offline installer, since in that case, the main PA installation page will be nothing but this page.